### PR TITLE
Feat/instructions

### PR DESCRIPTION
**Pull request by Victorafx**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -47,7 +47,8 @@ function App() {
                     <SearchBar />
                   </Route>
                   <Route path="/Contact" />
-                  <Route exact path="/" />
+                  <Route exact path="/" component={PictureDay} />
+                  <Route />
```
Comment (rocambille) [inline src/App.jsx]: this empty route looks useless, isn't it?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (Victorafx) [reply src/App.jsx]: Line 49? component contact will come later

Comment (rocambille) [reply src/App.jsx]: line 51, but you're right: line 49 is also useless. if component will come later, then add the route later ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (Victorafx) [reply src/App.jsx]: ok, exact, line 51 is useless

**Pull request merged**